### DISCUSSION TITLE
Figure is not showing up in tespy

### DISCUSSION THREAD

**Discussion by alinuman15** [oemof/tespy#387] [Q&A] [answered]
Dear community members: 
Although the following code asks to show the figure, it does not. Any idea ?
links['value'] = [val / links['value'][0] for val in links['value']]
fig = go.Figure(go.Sankey(
    arrangement="snap",
    textfont={"family": "Linux Libertine O"},
    node={
        "label": nodes,
        'pad': 11,
        'color': 'orange'},
    link=links))
fig.show()

**Comment by fwitte** [comment, marked as answer]
Hi,

on my machine this opens a new browser window and shows the plotly plot. Maybe the behaviour change with a different version of plotly. You could, for example, export the plot like here https://plotly.com/python/interactive-html-export/ and then open the html file with your browser.

Best

**Reply by alinuman15** [reply]
Thank you for replying. That is kind of lengthy way for plotting though, I will try to use something else   such as matplotlib.

**Reply by fwitte** [reply]
Well, the Sankey diagram of Matplotlib is not really looking too good… that is, why we chose plotly for this specific plot
